modules/imgcodecs/src/ios_conversions.mm

```diff
@@ -116,6 +118,8 @@ void UIImageToMat(const UIImage* image,
         if (!alphaExist)
             bitmapInfo = kCGImageAlphaNoneSkipLast |
                                 kCGBitmapByteOrderDefault;
+        else
+            m = cv::Scalar(0);
```

Comment:
Thanks.  You can't see it on the default diff on github, but if you expand the section above it you can see on line 102 that bitmapInfo is set to `kCGImageAlphaPremultipliedLast` as a default value.  Not the way I'd code this function, but I was trying to make the smallest change possible that fixed the bug.